cmd/build.go

```diff
@@ -0,0 +1,72 @@
+package cmd
+
+import (
+	"errors"
+	"io/ioutil"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/dominodatalab/forge/internal/buildjob"
+)
+
+var (
+	resourceName      string
+	resourceNamespace string
+
+	buildCmd = &cobra.Command{
+		Use:     "build",
+		Short:   "Launch a single OCI image build",
+		Long:    "fill in the details",
+		PreRunE: processResourceArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := buildjob.Config{
+				ResourceName:        resourceName,
+				ResourceNamespace:   resourceNamespace,
+				BrokerOpts:          brokerOpts,
+				PreparerPluginsPath: preparerPluginsPath,
+				EnableLayerCaching:  enableLayerCaching,
+				Debug:               debug,
+			}
+
+			job, err := buildjob.New(cfg)
+			if err != nil {
+				panic(err)
+			}
+			defer job.Cleanup()
+
+			if err := job.Run(); err != nil {
+				panic(err)
+			}
+		},
+	}
+)
+
+func processResourceArgs(cmd *cobra.Command, args []string) error {
+	var errMsgs []string
+
+	if resourceName == "" {
+		errMsgs = append(errMsgs, "'--resource' is required")
+	}
```

Comment:
You can't `MarkFlagRequired` it?

Reply:
I had that initially and then realized I needed some extra logic for `resourceNamespace` so that we try to load the svcaccount namespace from the pod when running in k8s. I can handle the two separately but it might result in an inconsistent error message. Let me review this again and see what I find.